data/language/en-US.yml

```diff
@@ -2179,3 +2179,5 @@ strings:
   2123: hp
   2124: kW
   2125: "{COLOUR WINDOW_2}Language:"
+
+  2227: "Trains reverse at signals"
```

Comment:
This can be reverted, as `en-GB` is used as fallback. (No rush, can be tackled with other review comments.)